docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Typos in below cell:</p><ul><li>"Sampling from distributions with multiple peaks with <s>a</s> standard MCMC methods can be difficult"</li><li>"explore the solution space<strong>,</strong> avoid<s>ing</s> getting trap in local minima and then apply <em>selection</em> to <em>probabilistically</em> keep reasonable solutions<strong>,</strong> while also keeping some diversity."</li><li>"Moving into the realm of Bayesian statistics<strong>,</strong> each individual is a point in the <em>posterior space </em><strong>and</strong> mutations can be done in several ways<strong>. A</strong> general solution is to use a<strong>n</strong> MCMC method (like Metropolis-Hastings or NUTS) and run many Markov chains in parallel. The <em>fitness</em> is given by the posterior <strong>distributions</strong>: points with low posterior density will be removed and points <strong>with</strong> high posterior density will be used as the starting point of a next round of Markov chains (<s>T</s><strong>t</strong>his step is known as <em>reweighting</em> in the SMC literature).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>